scripts/create-api-key.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Will we need Windows support for this script?

Reply:
Potentially, but if so, then we might come up with some follow-ups